ChatEventHandler.py

```diff
@@ -112,15 +111,7 @@ def typemethod(data):
 
 def format_msg(data):
     '''formats the msg text string properly'''
-    msg = ''
-
-    # gets the message list amount
-    list_count = len(data['message']['message'])
-
-    # loops though each message list entries
-    for counter in range(0, list_count):
-        # joins the message together
-        msg += str(data['message']['message'][counter]['text'])
+    msg = ''.join(item["text"] for item in data["message"]["message"])
```

Comment:
when i tested this with and without the space. 

* With the space:  between quotes it added an extra space between the joins
* Without the space: no extra space between the joins

Sere is the results with having the space for that line (top) and without the space (bottom)
![image](https://cloud.githubusercontent.com/assets/20742198/25607343/f6403884-2f0d-11e7-88e5-ab2f607f68e3.png)

This is why I removed the space to make the text formatting look better. the joins are before and after the emoji and @ mention